keylime/cloud_verifier_tornado.py

```diff
@@ -103,32 +163,33 @@ def get(self):
         was not found, it either completed successfully, or failed.  If found, the agent_id is still polling
         to contact the Cloud Agent.
         """
+        session = self.make_session(engine)
         rest_params = common.get_restful_params(self.request.uri)
         if rest_params is None:
-            common.echo_json_response(self, 405, "Not Implemented: Use /agents/ interface")
+            common.echo_json_response(
+                self, 405, "Not Implemented: Use /agents/ interface")
             return
 
         if "agents" not in rest_params:
             common.echo_json_response(self, 400, "uri not supported")
-            logger.warning('GET returning 400 response. uri not supported: ' + self.request.path)
+            logger.warning(
+                'GET returning 400 response. uri not supported: ' + self.request.path)
             return
 
         agent_id = rest_params["agents"]
 
         if agent_id is not None:
-            agent = self.db.get_agent(agent_id)
+            agent = session.query(VerfierMain).filter_by(agent_id=agent_id).one()
```

Comment:
Can we use exception here?

try:
session query
except sqlalchemy.orm.exc.NoResultFound as e:

Reply:
Otherwise use `one_or_none()`

Reply:
I think this might be related to a `500` error I'm getting when running the WebApp.  When the WebApp asks for an agent that isn't in the Verifier yet (but is in the Registrar), this block is throwing an exception.  Agree we either need to handle this exception or use `one_or_none()` (or both).

Reply:
Ok, I think there are two exceptions associated with this query: 1) one no entry is found, 2) multiple entries are found. `one_or_none()` will consume (1), but not the (2).

I think `one_or_none()` will be suffice.